xml/hwbp_registers.xml

```diff
@@ -1132,19 +1132,20 @@
     <register name="Interrupt Trigger" short="itrigger" address="0x7a1">
         This register is accessible as \RcsrTdataOne when \FcsrTdataOneType is 4.
 
-        This trigger may fire on any of the interrupts configurable in \Rmie
-        (described in the Privileged Spec) or the NMI. The interrupts to fire on are
-        configured by setting the same bit in \RcsrTdataTwo as would be set in
-        \Rmie to enable the interrupt.
+        For interrupts where it is enabled, this trigger fires when an interrupt
+        trap is taken from a privilege mode where the trigger is enabled.
```

Comment:
I wasn't sure what the first "it" is.  It seems like it refers to an interrupt but singular "it" doesn't match "interrupts" plural.  Then I wondered if it would be clearer to just remove everything before the comma.

Reply:
How about:
This trigger fires when an interrupt trap is taken from a privilege mode
where the trigger is enabled and that interrupt is also enabled in the
trigger.

Reply:
Now I understand.  There are a lot of enables here: the mode bits in tdata1, the interrupt number in tdata2, and the interrupt enables (both mie and mstatus.MIE).  I like something more like this tweak to the earlier language:

        For an interrupt where that interrupt number is enabled in tdata2, this trigger fires when an interrupt
        trap is taken from a privilege mode where the trigger is enabled.

That doesn't actually talk about the interrupt enables but that's implicit when you talk about taking a trap.

Reply:
I don't want to explicitly call out tdata2 here, because nmi is also an option. I agree there are a lot of enables. Maybe I just need to capitulate and make a numbered list.

Reply:
I made it even simpler. We don't need to talk about the mode at all in this sentence. It's already in the bits. The only real need is to indicate what the trigger is useful for. 2 paragraphs down (When the trigger matches...) already makes it clear that it only happens when a trap is actually taken. We don't need the parenthetical about interrupts being masked or whatever. The trigger fires when the trap is taken. Done.